index.html

```diff
@@ -142,8 +142,10 @@ <h2>Language:</h2>
     </div>
     <div class="row">
       <div id="copy-to-clipboard" class="text-end"><a href="#">Copy to clipboard</a></div>
-      <div style="display: none;" id="issue-promo">If your syntax is correct, create a
-        <a href="https://github.com/curlconverter/curlconverter/issues/new" target="_blank">new issue</a> <span style="display: none;" id="prefilled-issue"> or <a href="" target="_blank">prefilled with the current input</a></span>
+      <div style="display: none;" id="issue-promo">If your syntax is correct,
+        open a <a href="https://github.com/curlconverter/curlconverter/issues/new" target="_blank">new issue on GitHub</a>
+        <span id="prefilled-issue">or one <a href="https://github.com/curlconverter/curlconverter/issues/new" target="_blank">prefilled with the current input</a></span>
+        <span id="email-devs">or <a href="mailto:nickc@trillworks.com" target="_blank">email us</a></span>
```

Comment:
@NickCarneiro which email should I use here?

Ideally, if you can, could you please set up a dedicated `bugs@curlconverter.com` email that forwards these emails to us? I think you can do this [with Google Workspace](https://support.google.com/business/answer/9270657?hl=en#zippy=%2Cmanage-your-meetings-and-email%2Cmanage-your-team), but when I had a personal email address through google I remember it being quite expensive.